RPLCD/lcd.py

```diff
@@ -130,6 +130,7 @@ class CharLCD(object):
     # Init, setup, teardown
 
     def __init__(self, pin_rs=15, pin_rw=18, pin_e=16, pins_data=[21, 22, 23, 24],
+                       pin_backlight=None, backlight_active_low=False,
```

Comment:
I think we should change the name of `backlight_active_low` to `backlight_mode` to be in line with `numbering_mode`.

On line 111 we could then add a new enum:

```python
class BacklightMode(enum.Enum):
    active_high = 1
    active_low = 2
```

That makes it possible to create more obvious mode assignments:

```python
pin_backlight=None, backlight_mode=BacklightMode.active_high,
```